### ISSUE TITLE
oelint.vars.pnbpnusage should only apply to the URL part

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#75]
```
SRC_URI += "git://foo.org/baz.git;name=${PN}"
```

is a valid snippet and shouldn't raise a `oelint.vars.pnbpnusage` warning
